com.microsoft.mrtk.input/Assets/Prefabs/MRTK XR Rig.prefab

```diff
@@ -214,7 +216,7 @@ Transform:
   m_PrefabAsset: {fileID: 0}
   m_GameObject: {fileID: 2351505566903569413}
   m_LocalRotation: {x: 0, y: 0, z: 0, w: 1}
-  m_LocalPosition: {x: 0, y: 0, z: 0}
+  m_LocalPosition: {x: 0, y: 1.6, z: 0}
```

Comment:
Is this the camera offset object? Does this need to be baked into the prefab? For some reason, I thought this was applied at runtime

Reply:
This is because input simulation behaves like a Device-Relative-mode device, but we can't spin up a simulated XRInputSubsystem to actually tell the XROrigin that. When the XROrigin can't find a valid XRInputSubsystem, it does _nothing_, and so we have to do this "for it". An alternative solution would be to make the simulator floor-relative, but that requires much more effort!